README.md

```diff
@@ -27,9 +27,30 @@ $ mkdir -p $GOPATH/src/github.com/MailRuCloudSolutions
 $ cd $GOPATH/src/github.com/MailRuCloudSolutions
 $ git clone git@github.com:MailRuCloudSolutions/terraform-provider-mcs.git
 $ cd $GOPATH/src/github.com/MailRuCloudSolutions/terraform-provider-mcs
-$ make build
-...
-$ $GOPATH/bin/terraform-provider-mcs
+$ make build_darwin
```

Comment:
It looks like it would build OS X binary, shouldn't we divide this topic by OS platform?

Reply:
not sure, but okay i'll add a notice about other platforms as well

Reply:
comment added